webserver/templates/user/profile.html

```diff
@@ -67,8 +67,18 @@ <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
   {{ super() }}
   <script src="{{ url_for('static', filename='js/lib/jquery.timeago.js') }}"></script>
   <script type="text/javascript">
+    var offset = parseInt(new Date().getTimezoneOffset())*-1;
```

Comment:
This sounds like something that might be useful in more places than just the user profile for now. How about we move it to a javascript file as a separate function?
We don't have any custom javascript files in ListenBrainz yet (the import/scraper script is an exception because it requires the template engine).
You could copy the setup that we have in AcousticBrainz:
We use some javascript tools to compile javascript and less (css):
* The dependencies are at https://github.com/metabrainz/acousticbrainz-server/blob/master/package.json (@gentlecat or @mwiencek can tell you which dependencies we need for this, I think some of them may not be needed) 
* https://github.com/metabrainz/acousticbrainz-server/blob/master/gulpfile.js and https://github.com/metabrainz/acousticbrainz-server/blob/master/webserver/static/gulpfile.js are the build files. They should be able to be copied directly
* https://github.com/metabrainz/acousticbrainz-server/blob/master/webserver/static_manager.py is a helper which lets us import a file by its source name, and translate it to its "compiled" name
* https://github.com/metabrainz/acousticbrainz-server/blob/master/webserver/__init__.py#L61 is how we make this function available in templates
* https://github.com/metabrainz/acousticbrainz-server/blob/master/webserver/templates/base.html#L59 is an example of using it in templates